src/ui/hexedit.cc

```diff
@@ -654,9 +654,12 @@ void HexEdit::processMoveEvent(QKeyEvent *event) {
 
 void HexEdit::processSelectionChangeEvent(QKeyEvent *event)
 {
+  if (event->matches(QKeySequence::Cancel)) {
```

Comment:
do not compile on jenkins, because ::Cancel it doens;'t exists on this qt version